joblib/test/test_logger.py

```diff
@@ -10,11 +10,9 @@
 
 from joblib.logger import PrintTime
 
-try:
-    # Python 2/Python 3 compat
-    unicode('str')
-except NameError:
-    unicode = lambda s: s
+
+def unicode(s):
```

Comment:
Maybe raise a deprecation warning. This function is useless and should never be used.

Reply:
As this function is in a test file, should we bother use a deprecation warning?

Reply:
Yep I don't think a deprecation warning is necessary in `test_*.py` files